src/frameworks/index.ts

```diff
@@ -59,8 +62,9 @@ export const frameworks: Framework[] = [
   new JekyllFramework(),
   new GeneralFramework(),
 
-  // Vue SFC should be the last one
+  // Vue SFC and FluentVue SFC should be the last ones
   new VueSFCFramework(),
+  new FluentVueSFCFramework(),
```

Comment:
I don't know why Vue SFC should be the last. But added FluentVue SFC here too as it uses the same functionality as Vue SFC